app/main.py

```diff
@@ -1 +1,57 @@
-#  write your code here
+from typing import List
+
+
+class Animal:
+    def __init__(self,
```

Comment:
Pass there variables with an indent like [here ](https://stackoverflow.com/questions/24988162/define-functions-with-too-many-arguments-to-abide-by-pep8-standard)